lib/percy/capybara/client.rb

```diff
@@ -62,7 +67,10 @@ def failed?
       end
 
       def initialize_loader(options = {})
-        if sprockets_environment && sprockets_options
+        if custom_loader?
+          Percy.logger.debug { 'Using a custom loader to discover assets.' }
+          @custom_loader.new(options)
```

Comment:
Use `custom_loader` here instead of `@custom_loader` (just a personal style preference to prefer using accessors that will raise errors if spelled wrong instead of instance vars that will silently return `nil`).